Queueomatic/Client/Pages/Room.razor

```diff
@@ -0,0 +1,72 @@
+﻿@*there needs to be a check if the room owner is connecting when participantname is omitted*@
+@page "/room/{RoomId}"
+@page "/room/{RoomId}/{ParticipantName}"
+@using Queueomatic.Shared.DTOs
+
+<LogoComponent />
+<h1 class="text-center roomText mt-5">Welcome to @RoomName!</h1>
+
+<div class="container">
+    <div class="row py-4">
+        <div class="col-12 d-flex flex-column align-items-center justify-content-center mb-5">
+            <p class="header">Currently helping</p>
+            <div class="d-flex flex-column ">
+                <Dropzone Items="ActiveParticipants" TItem="string" Accepts="(current, hoveredItem) => hoveredItem == null" AllowsDrag="s => CanMove(s)" OnItemDrop="@(user => UpdateUser(user, StatusDto.Idling))" Class="plk-dd-dropzone h-100">
+                    <ChildContent>
+                        <div class="card box">
+                            <div class="card-body">
+                                <h5 class="text-center">@context</h5>
+                            </div>
+                        </div>
+                    </ChildContent>
+                </Dropzone>
+            </div>
+        </div>
+        <div class="col d-flex flex-column align-items-center justify-content-center mb-5">
+            <p class="header">Idling</p>
```

Comment:
The idling list should be under waiting for help in mobile view. It is more important to see them than who is idling.